src/main/java/com/connexta/ddf/catalog/direct/CatalogMethods.java

```diff
@@ -427,7 +425,7 @@ private Object query(Map<String, Object> params) {
 
   private Map<String, Object> getMetacardInfo(Metacard metacard) {
     return new ImmutableMap.Builder<String, Object>()
-        .put("metacard", ImmutableMap.of("properties", this.metacard2map(metacard)))
```

Comment:
Breaking change -- `properties` -> `attributes` 
will need to be updated in clients.